src/_vars.scss

```diff
@@ -16,15 +17,15 @@ $secondary-red: #b53228;
 
 $alternative-red: #ff0000;
 
-$neutral-white: #ffffff;
-$neutral-white-opacity80: #ffffffcd;
-$neutral-white-opacity50: #ffffff80;
-$neutral-white-opacity30: #ffffff4d;
-$neutral-light-gray: #ebeaea;
-$neutral-light-gray2: #dddddd;
-$neutral-gray: #808080;
-$neutral-gray2: #cccccb;
-$neutral-gray3: #adadad;
+$neutral-white: #ffffff; //Setting: header buttons, light event cards, person profile(both pub and priv), about and help card header text, people search icon,
```

Comment:
Please put each comment above its color, unless it is very short, so there are no lines over 80 characters long.

Reply:
We don't actually need these comments any more, so I'll just remove them...